### ISSUE TITLE
rename plugin

### ISSUE THREAD

**Issue by z3t0** [deathcap/voxel-fluid#2]
it may be a good idea to rename this to something like voxel-fluid-register for clarity

**Comment by deathcap** [comment]
Agreed the naming is suboptimal, maybe something like voxel-registry-fluid or voxel-fluid-registry (like [voxel-registry](https://github.com/voxel/voxel-registry)), or alternatively merge into voxel-registry itself? But on the other hand, I would like there to be abstracted fluid flow logic somewhere, it could live in this plugin (voxel-fluid), so the name is mostly a reflection of its incompleteness (https://github.com/voxel/ideas/issues/1 Water (fluids/liquids) )

**Comment by z3t0** [comment]
@deathcap I definitely think keeping the plugins separate is a good idea, and i guess the naming convention applies itself more to items and blocks
